microsoft/FluidStorybook
# Source code viewer build process and webpack optimizations

This PR does the following:

1. Creates a source viewer build process as a standalone script. Generates rawSources.json file and puts it in /public.
2. Updates package.json build scripts to call this new source viewer build script.
3. Fixes a minor bug with the drop-down in the source viewer.
4. Cleans up older webpack loader code that originally generated the rawSources.json file.
5. Modifies the Fluid Playground link that displays in the upper-left corner
6. Updates Fluid packages to 0.26.0
7. Added new source viewer build script into webpack so it 'll run anytime webpacks rebuilds /src

Does the CI/CD process still work with the latest move from SWA to blob storage? Wasn't sure if the same blob storage was being used for both or if things changed there.

## Verification Steps

Clone this branch and run:

1. `npm start` (app should load fine and you should see a rawSources.json in /public that's been updated).
2. `npm run build:hugo` (storybook-static folder that's generated should have a rawSources.json file). I normally run `npx http-server` from within the `storybook-static` folder to try it.

Thanks to Captain Jokiel (@skylerjokiel) for the initial look at optimizations, the main.js webpack contribution, and for triggering the idea for this new build/optimization approach.
